ArduCopter/GCS_Mavlink.cpp

```diff
@@ -71,7 +71,7 @@ NOINLINE void Copter::send_heartbeat(mavlink_channel_t chan)
 #endif
 
     // we are armed if we are not initialising
-    if (motors->armed()) {
+    if (motors != nullptr && motors->armed()) {
```

Comment:
Because of adding delay for magnetometer and barometer discovery in AP_BoardConfig, I get SEGFAULT since object motors doesn't created.